tests/compute/subresources/vmi.go

```diff
@@ -59,7 +60,7 @@ var _ = compute.SIGDescribe("VirtualMachineInstance subresource", func() {
 		BeforeEach(func() {
 			var err error
 			vmi := libvmifact.NewCirros()
-			vm = libvmi.NewVirtualMachine(vmi, libvmi.WithRunning())
+			vm = libvmi.NewVirtualMachine(vmi, libvmi.WithRunStrategy(v1.RunStrategyAlways))
```

Comment:
The comment is not meant to be on this line, it is generic.

I would say we miss-place these tests. The `subresources` is technical detail but the real meaning or context of these tests is life cycle.